app/config/schema.json

```diff
@@ -0,0 +1,593 @@
+{
+  "type": "object",
+  "title": "Fundraising app configuration",
+  "properties": {
+    "db": {
+      "type": "object",
+      "title": "Database settings",
+      "description": "Keys and values of all properties here depend on the database driver",
+      "properties": {
+        "driver": {
+          "type": "string",
+          "minLength": 1,
+          "title": "Database driver name.",
+          "description": "A Doctrine database driver, e.g. mysql or sqlite",
+          "default": ""
+        },
+        "user": {
+          "type": "string",
+          "minLength": 1,
+          "title": "User name",
+          "default": ""
+        },
+        "password": {
+          "type": "string",
+          "minLength": 1,
+          "title": "Password",
+          "default": ""
+        },
+        "dbname": {
+          "type": "string",
+          "minLength": 1,
+          "title": "Database name",
+          "default": ""
+        },
+        "host": {
+          "type": "string",
+          "minLength": 1,
+          "title": "Host name",
+          "description": "Database host name",
+          "default": ""
+        },
+        "port": {
+          "type": "integer",
+          "title": "Database port",
+          "default": ""
+        },
+        "driverOptions": {
```

Comment:
I've edited and expanded the description and explained what to put in there and why. Now that I look at this I think this should not be configurable but should be placed in the factory, depending on the driver name. But that's a different PR.